Public/Src/Engine/Dll/BuildXL.Engine.dsc

```diff
@@ -39,6 +39,7 @@ namespace Engine {
             importFrom("BuildXL.Cache.MemoizationStore").Interfaces.dll,
             importFrom("BuildXL.Pips").dll,
             importFrom("BuildXL.Ide").Generator.dll,
+            importFrom("BuildXL.Ide").Generator.Old.dll,

```

Comment:
This `Generator.Old` project is a verbatim copy of the old generator project.  In git those files all appear as "new", but it's safe to ignore them for the purpose of this PR.

The reason why I copied the old project into a new folder instead of putting the new generator into a new folder is so that `git diff` shows the changes between the new and the old generator more clearly.

The old generator remains the default until the new one is vetted more thoroughly.